### PR TITLE
[14.0][mig]html_image_url_extractor

### PR DESCRIPTION
**Pull request by hkapatel-initos**
Migration to 14.0

### CODE REVIEW

**Review by fshah-initos — APPROVED** [review]
LGTM.  No major change

**Comment by hkapatel-initos** [comment]
@victoralmau  it would be nice if could you review it here.

File: html_image_url_extractor/i18n/am.po [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Última actualización por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Última actualización en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/am.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/ar.po [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "أنشئ بواسطة"
+
+#~ msgid "Created on"
+#~ msgstr "أنشئ في"
+
+#~ msgid "Last Updated by"
+#~ msgstr "آخر تحديث بواسطة"
+
+#~ msgid "Last Updated on"
+#~ msgstr "آخر تحديث في"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/ar.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/bg.po [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Създадено от"
+
+#~ msgid "Created on"
+#~ msgstr "Създадено на"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Последно обновено от"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Последно обновено на"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/bg.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/bs.po [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Kreirao"
+
+#~ msgid "Created on"
+#~ msgstr "Kreirano"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Zadnji ažurirao"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Zadnje ažurirano"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/bs.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/cs.po [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Vytvořil(a)"
+
+#~ msgid "Created on"
+#~ msgstr "Vytvořeno"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Naposled upraveno"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Naposled upraveno"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/cs.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/da.po [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Oprettet af"
+
+#~ msgid "Created on"
+#~ msgstr "Oprettet den"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Sidst opdateret af"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Sidst opdateret den"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/da.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/el_GR.po [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Δημιουργήθηκε από "
+
+#~ msgid "Created on"
+#~ msgstr "Δημιουργήθηκε στις"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Τελευταία ενημέρωση από"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Τελευταία ενημέρωση στις"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/el_GR.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/en_GB.po [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Created by"
+
+#~ msgid "Created on"
+#~ msgstr "Created on"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Last Updated by"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Last Updated on"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/en_GB.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_AR.po [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Última actualización realizada por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Última actualización el"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_AR.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_CL.po [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Última actualización de"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Última actualización en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_CL.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_CO.po [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Actualizado por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Actualizado"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_CO.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_CR.po [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Ultima actualización por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Ultima actualización en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_CR.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_DO.po [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Última actualización de"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Última actualización en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_DO.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_EC.po [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Última actualización de"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Última actualización en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_EC.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_MX.po [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Ultima actualizacion por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Ultima actualización realizada"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_MX.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_PE.po [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Actualizado última vez por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Ultima Actualización"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_PE.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_PY.po [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Ultima actualización por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Ultima actualización en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_PY.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/es_VE.po [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Última actualización realizada por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Ultima actualizacion en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/es_VE.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/et.po [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Loonud"
+
+#~ msgid "Created on"
+#~ msgstr "Loodud"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Viimati uuendatud"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Viimati uuendatud"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/et.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/eu.po [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Nork sortua"
+
+#~ msgid "Created on"
+#~ msgstr "Created on"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Last Updated by"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Last Updated on"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/eu.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/fa.po [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "ایجاد شده توسط"
+
+#~ msgid "Created on"
+#~ msgstr "ایجاد شده در"
+
+#~ msgid "Last Updated by"
+#~ msgstr "آخرین به روز رسانی توسط"
+
+#~ msgid "Last Updated on"
+#~ msgstr "آخرین به روز رسانی در"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/fa.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/fi.po [hunk 22]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Luonut"
+
+#~ msgid "Created on"
+#~ msgstr "Luotu"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Viimeksi päivittänyt"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Viimeksi päivitetty"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/fi.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/fr.po [hunk 23]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Créé par"
+
+#~ msgid "Created on"
+#~ msgstr "Créé le"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Dernière mise à jour par"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Dernière mise à jour le"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/fr.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/fr_CA.po [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Créé par"
+
+#~ msgid "Created on"
+#~ msgstr "Créé le"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Dernière mise à jour par"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Dernière mise à jour le"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/fr_CA.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/fr_CH.po [hunk 25]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Créé par"
+
+#~ msgid "Created on"
+#~ msgstr "Créé le"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Modifié par"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Modifié le"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/fr_CH.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/gl.po [hunk 26]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Creado por"
+
+#~ msgid "Created on"
+#~ msgstr "Creado en"
+
+#~ msgid "Last Updated by"
+#~ msgstr "ültima actualización por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Última actualización en"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/gl.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/he.po [hunk 27]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "נוצר על ידי"
+
+#~ msgid "Created on"
+#~ msgstr "נוצר ב-"
+
+#~ msgid "Last Updated by"
+#~ msgstr "עודכן לאחרונה על ידי"
+
+#~ msgid "Last Updated on"
+#~ msgstr "עודכן לאחרונה על"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/he.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/hr.po [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Kreirao"
+
+#~ msgid "Created on"
+#~ msgstr "Kreirano"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Zadnji ažurirao"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Zadnje ažuriranje"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/hr.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/hr_HR.po [hunk 29]
```diff
@@ -0,0 +1,38 @@
...
+#~ msgstr "Kreirao"
+
+#~ msgid "Created on"
+#~ msgstr "Kreirano"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Zadnji ažurirao"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Zadnje ažurirano"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/hr_HR.po:38]:
```suggestion
```

File: html_image_url_extractor/i18n/hu.po [hunk 30]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Készítette"
+
+#~ msgid "Created on"
+#~ msgstr "Létrehozás dátuma"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Utoljára frissítve, által"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Utoljára frissítve "
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/hu.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/id.po [hunk 31]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Dibuat oleh"
+
+#~ msgid "Created on"
+#~ msgstr "Dibuat pada"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Diperbaharui oleh"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Diperbaharui pada"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/id.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/ja.po [hunk 32]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "作成者"
+
+#~ msgid "Created on"
+#~ msgstr "作成日"
+
+#~ msgid "Last Updated by"
+#~ msgstr "最終更新者"
+
+#~ msgid "Last Updated on"
+#~ msgstr "最終更新日"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/ja.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/ko.po [hunk 33]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "작성자"
+
+#~ msgid "Created on"
+#~ msgstr "작성일"
+
+#~ msgid "Last Updated by"
+#~ msgstr "최근 갱신한 사람"
+
+#~ msgid "Last Updated on"
+#~ msgstr "최근 갱신 날짜"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/ko.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/lt.po [hunk 34]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Sukūrė"
+
+#~ msgid "Created on"
+#~ msgstr "Sukurta"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Paskutinį kartą atnaujino"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Paskutinį kartą atnaujinta"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/lt.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/lt_LT.po [hunk 35]
```diff
@@ -0,0 +1,38 @@
...
+#~ msgstr "Sukūrė"
+
+#~ msgid "Created on"
+#~ msgstr "Sukurta"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Paskutinį kartą atnaujino"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Paskutinį kartą atnaujinta"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/lt_LT.po:38]:
```suggestion
```

File: html_image_url_extractor/i18n/lv.po [hunk 36]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Izveidoja"
+
+#~ msgid "Created on"
+#~ msgstr "Izveidots"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Pēdējo reizi atjaunoja"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Pēdējās izmaiņas"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/lv.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/mk.po [hunk 37]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Креирано од"
+
+#~ msgid "Created on"
+#~ msgstr "Креирано на"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Последно ажурирање од"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Последно ажурирање на"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/mk.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/mn.po [hunk 38]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Үүсгэгч"
+
+#~ msgid "Created on"
+#~ msgstr "Үүсгэсэн"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Сүүлийн засвар хийсэн"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Сүүлийн засвар хийсэн огноо"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/mn.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/nb.po [hunk 39]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Opprettet av"
+
+#~ msgid "Created on"
+#~ msgstr "Opprettet den"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Sist oppdatert av"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Sist oppdatert"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/nb.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/nb_NO.po [hunk 40]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Laget av"
+
+#~ msgid "Created on"
+#~ msgstr "Laget den"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Sist oppdatert av"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Sist oppdatert den"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/nb_NO.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/nl.po [hunk 41]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Aangemaakt door"
+
+#~ msgid "Created on"
+#~ msgstr "Aangemaakt op"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Laatst bijgewerkt door"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Laatst bijgewerkt op"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/nl.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/nl_BE.po [hunk 42]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Gemaakt door"
+
+#~ msgid "Created on"
+#~ msgstr "Gemaakt op"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Laatst bijgewerkt door"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Laatst bijgewerkt op"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/nl_BE.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/pl.po [hunk 43]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Utworzone przez"
+
+#~ msgid "Created on"
+#~ msgstr "Utworzono"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Ostatnio modyfikowane przez"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Ostatnia zmiana"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/pl.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/pt.po [hunk 44]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Criado por"
+
+#~ msgid "Created on"
+#~ msgstr "Criado em"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Atualizado pela última vez por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Atualizado pela última vez em"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/pt.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/pt_BR.po [hunk 45]
```diff
@@ -0,0 +1,38 @@
...
+#~ msgstr "Criado por"
+
+#~ msgid "Created on"
+#~ msgstr "Criado em"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Última atualização por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Última atualização em"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/pt_BR.po:38]:
```suggestion
```

File: html_image_url_extractor/i18n/pt_PT.po [hunk 46]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Criado por"
+
+#~ msgid "Created on"
+#~ msgstr "Criado em"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Atualizado pela última vez por"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Atualizado pela última vez em"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/pt_PT.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/ro.po [hunk 47]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Creat de"
+
+#~ msgid "Created on"
+#~ msgstr "Creat la"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Ultima actualizare făcută de"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Ultima actualizare la"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/ro.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/ru.po [hunk 48]
```diff
@@ -0,0 +1,38 @@
...
+#~ msgstr "Создано"
+
+#~ msgid "Created on"
+#~ msgstr "Создан"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Последний раз обновлено"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Последний раз обновлено"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/ru.po:38]:
```suggestion
```

File: html_image_url_extractor/i18n/sk.po [hunk 49]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Vytvoril"
+
+#~ msgid "Created on"
+#~ msgstr "Vytvorené"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Naposledy upravil"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Naposledy upravené"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/sk.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/sl.po [hunk 50]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Ustvaril"
+
+#~ msgid "Created on"
+#~ msgstr "Ustvarjeno"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Zadnjič posodobil"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Zadnjič posodobljeno"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/sl.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/sr.po [hunk 51]
```diff
@@ -0,0 +1,28 @@
...
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+
+#. module: html_image_url_extractor
+#: model:ir.model,name:html_image_url_extractor.model_ir_fields_converter
+msgid "Fields Converter"
+msgstr ""
+
+#~ msgid "Created on"
+#~ msgstr "Kreiran"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/sr.po:28]:
```suggestion
```

File: html_image_url_extractor/i18n/sr@latin.po [hunk 52]
```diff
@@ -0,0 +1,38 @@
...
+#~ msgstr "Kreirao"
+
+#~ msgid "Created on"
+#~ msgstr "Kreiran"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Zadnja izmjena"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Zadnja izmjena"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/sr@latin.po:38]:
```suggestion
```

File: html_image_url_extractor/i18n/sv.po [hunk 53]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Skapad av"
+
+#~ msgid "Created on"
+#~ msgstr "Skapad den"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Senast uppdaterad av"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Senast uppdaterad"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/sv.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/th.po [hunk 54]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "สร้างโดย"
+
+#~ msgid "Created on"
+#~ msgstr "สร้างเมื่อ"
+
+#~ msgid "Last Updated by"
+#~ msgstr "อัพเดทครั้งสุดท้ายโดย"
+
+#~ msgid "Last Updated on"
+#~ msgstr "อัพเดทครั้งสุดท้ายเมื่อ"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/th.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/tr_TR.po [hunk 55]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Oluşturan"
+
+#~ msgid "Created on"
+#~ msgstr "Oluşturulma tarihi"
+
+#~ msgid "Last Updated by"
+#~ msgstr "En son güncelleyen "
+
+#~ msgid "Last Updated on"
+#~ msgstr "En son güncelleme tarihi"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/tr_TR.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/uk.po [hunk 56]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Створив"
+
+#~ msgid "Created on"
+#~ msgstr "Дата створення"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Востаннє оновив"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Останнє оновлення"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/uk.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/vi.po [hunk 57]
```diff
@@ -0,0 +1,36 @@
...
+#~ msgstr "Được tạo bởi"
+
+#~ msgid "Created on"
+#~ msgstr "Được tạo vào"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Last Updated by"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Cập nhật lần cuối vào"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/vi.po:36]:
```suggestion
```

File: html_image_url_extractor/i18n/vi_VN.po [hunk 58]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "Tạo bởi"
+
+#~ msgid "Created on"
+#~ msgstr "Tạo vào"
+
+#~ msgid "Last Updated by"
+#~ msgstr "Cập nhật lần cuối bởi"
+
+#~ msgid "Last Updated on"
+#~ msgstr "Cập nhật lần cuối vào"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/vi_VN.po:37]:
```suggestion
```

File: html_image_url_extractor/i18n/zh_CN.po [hunk 59]
```diff
@@ -0,0 +1,38 @@
...
+#~ msgstr "创建者"
+
+#~ msgid "Created on"
+#~ msgstr "创建时间"
+
+#~ msgid "Last Updated by"
+#~ msgstr "最后更新者"
+
+#~ msgid "Last Updated on"
+#~ msgstr "上次更新日期"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/zh_CN.po:38]:
```suggestion
```

File: html_image_url_extractor/i18n/zh_TW.po [hunk 60]
```diff
@@ -0,0 +1,37 @@
...
+#~ msgstr "建立者"
+
+#~ msgid "Created on"
+#~ msgstr "建立於"
+
+#~ msgid "Last Updated by"
+#~ msgstr "最後更新："
+
+#~ msgid "Last Updated on"
+#~ msgstr "最後更新於"
```
Comment (victoralmau) [inline html_image_url_extractor/i18n/zh_TW.po:37]:
```suggestion
```

File: html_image_url_extractor/readme/DESCRIPTION.rst [hunk 61]
```diff
@@ -0,0 +1,6 @@
+This module includes a method that extracts image URLs from any chunk of HTML,
+in appearing order.
+
+It is useful for modules such as `website_blog_excerpt_img
+<https://github.com/OCA/website/tree/9.0/website_blog_excerpt_img>`_: blog list shows the first
```
Comment (victoralmau) [inline html_image_url_extractor/readme/DESCRIPTION.rst]:
```suggestion
<https://github.com/OCA/website/tree/14.0/website_blog_excerpt_img>`_: blog list shows the first
```

File: html_image_url_extractor/static/description/index.html [hunk 62]
```diff
@@ -0,0 +1,456 @@
...
+<p>This module includes a method that extracts image URLs from any chunk of HTML,
+in appearing order.</p>
+<p>It is useful for modules such as <a class="reference external" href="https://github.com/OCA/website/tree/9.0/website_blog_excerpt_img">website_blog_excerpt_img</a>: blog list shows the first
```
Comment (victoralmau) [inline html_image_url_extractor/static/description/index.html]:
```suggestion
<p>It is useful for modules such as <a class="reference external" href="https://github.com/OCA/website/tree/14.0/website_blog_excerpt_img">website_blog_excerpt_img</a>: blog list shows the first
```

File: html_image_url_extractor/tests/test_extractor.py [hunk 63]
```diff
@@ -0,0 +1,56 @@
...
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+from odoo.tools import mute_logger
+
+
+class ExtractorCase(TransactionCase):
+    def setUp(self):
+        super(ExtractorCase, self).setUp()
```
Comment (victoralmau) [inline html_image_url_extractor/tests/test_extractor.py]:
```suggestion
        super().setUp()
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Good job, only some little changes related to purge translations.

**Comment by hkapatel-initos** [comment]
> Good job, only some little changes related to purge translations.

Thanks for your suggestion some changes are done, as per my experience we don't update the translation.

**Comment by victoralmau** [comment]
> > Good job, only some little changes related to purge translations.
> 
> Thanks for your suggestion some changes are done, as per my experience we don't update the translation.

Ok, thanks, although translations purge not appear in migration guideline is a better work because transbot not do it.

**Review by victoralmau — APPROVED** [review]

**Comment by hkapatel-initos** [comment]
@pedrobaeza  It would be great if you could try to merge it.

**Pull request merged**